src/genie/libs/parser/iosxe/show_wlan.py

```diff
@@ -104,3 +103,748 @@ def cli(self, output=None):
+            # EoGRE Mobility Handoff error                                    : 0
+            r"\s+EoGRE Mobility Handoff error\s+:\s+(?P<eogre_mobility_handoff_error>\d+)\n+"
+            # IP Update timeout                                               : 0
+            r"\s+IP Update timeout\s+:\s+(?P<ip_update_timeout>\d+)\n+"
+            # L3 VLAN Override connection timeout                         : 0
+            r"\s+L3 VLAN Override connection timeout\s+:\s+(?P<connection_timeout_l3vlan_override>\d+)\n+"
+            # Mobility peer delete                                            : 0
+            r"\s+Mobility peer delete\s+:\s+(?P<mobility_peer_delete>\d+)\n+"
+            # NACK IFID mismatch                                              : 0
+            r"\s+NACK IFID mismatch\s+:\s+(?P<nack_ifid_mismatch>\d+)\n+"
+        )
+
+        wlan_obj = {}
+
+        if re.search(wlan_capture, out, re.MULTILINE):
+            search = re.search(wlan_capture, out, re.MULTILINE)
+            group = search.groupdict()
+
+            # convert str to int
+            group["id"] = int(group["id"])
+
+            new_group = {"wlan": group}
+            wlan_obj.update(new_group)
+
+        if re.search(client_stats_capture, out, re.MULTILINE):
+            search = re.search(client_stats_capture, out, re.MULTILINE)
+            group = search.groupdict()
+
+            for item in group:
+                # convert str to int
+                group[item] = int(group[item])
+
+            new_group = {"client_stats": group}
+            wlan_obj.update(new_group)
+
+        if re.search(client_delete_capture, out, re.MULTILINE):
+            search = re.search(client_delete_capture, out, re.MULTILINE)
+            group = search.groupdict()
+
+            for item in group:
+                # convert str to int
+                group[item] = int(group[item])
+
+            new_group = {"client_delete": group}
+
+            key_list = [
+                "aaa",
+                "anchor",
+                "ap_initiated_delete",
+                "client",
+                "connection_timeout",
+                "dot11",
+                "dot11v",
+                "eap",
+                "eogre",
+                "fqdn",
+                "mac",
+                "mobility",
+                "nack",
+                "policy",
+                "qos",
+                "wpa",
+                "wired",
+            ]
+
+            for key in key_list:
+                new_key_group = {key: {}}
+
+                for item in new_group["client_delete"].copy():
+                    format_key = "^{key}_".format(key=key)
+                    # if the key from key_list is found in item
+                    if re.search(format_key, item):
+                        # replace the key and update with new_dict
+                        new_key = re.sub(f"^{key}_", "", item)
```

Comment:
sorry, I found another f-string. please change this as well.

Reply:
AH! No my bad! I just dropped in the `format_key` variable I made above